jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -8,12 +8,10 @@
 
 @pytest.mark.filterwarnings('ignore:No observer defined on WCS')
 def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
-    app = cubeviz_helper.app
-    dc = app.data_collection
-    app.add_data(spectrum1d_cube, 'test[FLUX]')
-    app.add_data_to_viewer('flux-viewer', 'test[FLUX]')
+    dc = cubeviz_helper.app.data_collection
+    cubeviz_helper.load_data(spectrum1d_cube, data_label='test')
```

Comment:
Since I am updating this test anyway, I took the liberty to make it more realistic in loading like a user would.